frontend/src/libs/components/location-input/location-input.tsx

```diff
@@ -19,6 +22,7 @@ type Properties<T extends FieldValues> = {
   name: FieldPath<T>;
   placeholder?: string;
   isDisabled?: boolean;
+  onChange?: ((place: google.maps.LatLngLiteral) => void) | undefined;
```

Comment:
Is optional props not enough?
```suggestion
  onChange?: (place: google.maps.LatLngLiteral) => void;
```